onnxruntime/core/framework/tensorprotoutils.h

```diff
@@ -122,6 +125,11 @@ inline bool HasElemType(const ONNX_NAMESPACE::TypeProto_SparseTensor& ten_proto)
   return ten_proto.elem_type() != ONNX_NAMESPACE::TensorProto::UNDEFINED;
 }
 
+inline bool HasName(const ONNX_NAMESPACE::SparseTensorProto& ten_proto) {
+  return ten_proto.values().has_name(); // XXX
```

Comment:
>XXX [](start = 43, length = 3)

Remove this?

Reply:
There are 11 places like this in this file. proto3 does not have has_*() functions, we wrap those calls into helpers so when we know how to deal with them we can do so in one place.

---
In reply to: [512312964](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r512312964) [](ancestors = 512312964)